frontend/src/components/NewRunParametersV2.test.tsx

```diff
@@ -281,6 +281,197 @@ describe('NewRunParametersV2', () => {
     });
   });
 
+  it('test convertInput function for double type with default value', () => {
+    const handleParameterChangeSpy = jest.fn();
+    const props = {
+      titleMessage: 'default Title',
+      pipelineRoot: 'defalut pipelineRoot',
+      specParameters: {
+        doubleParam: {
+          parameterType: ParameterType_ParameterTypeEnum.NUMBER_DOUBLE,
+          defaultValue: 1.23,
+        },
+      },
+      handlePipelineRootChange: jest.fn(),
+      handleParameterChange: handleParameterChangeSpy,
+    };
+    render(<NewRunParametersV2 {...props} />);
+
+    const doubleParam = screen.getByDisplayValue('1.23');
+    fireEvent.change(doubleParam, { target: { value: '4.56' } });
+    expect(handleParameterChangeSpy).toHaveBeenCalledTimes(1);
+    expect(handleParameterChangeSpy).toHaveBeenLastCalledWith({
+      doubleParam: 4.56,
+    });
```

Comment:
For new tests: Please also check for the existence of display value 4.56. Just like we mentioned in https://github.com/kubeflow/pipelines/pull/7991/files#r916159217